src/propagate-job.ts

```diff
@@ -90,19 +93,40 @@ export async function run(): Promise<void> {
         type: 2,
       };
 
-      // We populate the transactions we will use in our bundles
+      const _connectors: string[] = [];
```

Comment:
I can't leave a comment in the options line, but if i recall correctly `propagate` can be quite a expensive call. Are you alright leaving the `options` object `gasLimit` field as `10_000_000`?